common/src/main/java/io/seata/common/util/CollectionUtils.java

```diff
@@ -277,4 +279,19 @@ public static String encodeMap(Map<String, String> map) {
             }
         }
     }
+
+    public static <T> List<List<T>> cutData(List<T> list, int n) {
+        if (isEmpty(list) || n <= 0) {
+            throw new ShouldNeverHappenException("list is empty or cut target size is zero");
+        }
+        int targetSize = (list.size() + n - 1) / n;
+
+        List<List<T>> resp = new ArrayList<>(targetSize);
+
+        for (int i = 0; i < targetSize; i ++) {
+            resp.add(list.subList(i * n, Math.min(i * n + n, list.size())));
+        }
+
+        return resp;
+    }
```

Comment:
`List<List<T>> parts = Lists.partition(list, n);` is a good idea?

Reply:
yes, i will modify late.